packages/adapter/README.md

```diff
@@ -22,13 +22,16 @@ This adapter interacts directly with the snap, so Metamask Flask needs to be ins
 ```js
 import * as Filsnap from 'filsnap-adapter'
 
-const hasFlask = await Filsnap.hasFlask()
+const hasFlask = await Filsnap.FilsnapAdapter.hasFlask()
 if (!hasFlask) {
```

Comment:
```js
import { FilsnapAdapter } from 'filsnap-adapter'
```

can we do this instead to make everything a bit smaller

Reply:
good call! updated 👍🏼